vscode-trace-webviews/src/trace-explorer/available-views/index.tsx

```diff
@@ -1,7 +1,13 @@
 import * as React from 'react';
 import ReactDOM from 'react-dom/client';
+import { Messenger } from 'vscode-messenger-webview';
 import './index.css';
 import TraceExplorerViewsWidget from './vscode-trace-explorer-views-widget';
 
+// @ts-expect-error "will be available when running the app"
+export const vscode = acquireVsCodeApi();
```

Comment:
I'm curious: is it useful to export this?  If not, `acquireVsCodeApi()` could be used directly when creating a new `Messenger` object below.

Reply:
It is needed to be able to access vscode APIs in the webview. We used to do it in the HTML that we create for each webview but that was not sufficient place when introducing the messenger.